### ISSUE TITLE
[AVM Module Issue]: Can avm.res.web.connection help with setting the correct parameter values?

### ISSUE THREAD

**Issue by TWolversonReply** [Azure/bicep-registry-modules#2585]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Issue Type?

Feature Request

### Module Name

avm/res/web/connection

### (Optional) Module Version

0.2.0

### Description

One of the biggest challenges I have with using Logic Apps is that there is no documentation for what parameters are required to set up any particular API connection. [The documentation](https://github.com/MicrosoftDocs/azure-docs/issues/122783)'s suggestion of using the REST API to find the parameter schema is really not satisfactory.

Can the verified module provide the extra intelligence to know what parameters are required per connection? This would be a great leap forward in usability and productivity.

**Comment by matebarabas** [comment]
Note: this module is currently orphaned. https://github.com/Azure/Azure-Verified-Modules/issues/767

**Comment by AlexanderSehr** [comment]
Agreed, the UDTs for this module should be added. Unfortunately, as the module is currently orphaned, it may take some time to implement the contribution.

**Comment by abhishekaryams** [comment]
I CONFIRM I WISH TO OWN THIS AVM MODULE AND UNDERSTAND THE REQUIREMENTS AND DEFINITION OF A MODULE OWNER

**Comment by abhishekaryams** [comment]
@TWolversonReply i agree that it need more documentation will help. my suggestion is to
Add more e2e test scenarios for popular connectors, which auto-generate README examples. Candidates:

Service Bus (connection string + MSI variants)
SQL Server
Office 365 / Outlook
SharePoint Online
Azure Key Vault
Azure Event Hubs
what is your thoughts for the same.
